overrides/extension-override.json

```diff
@@ -147,6 +147,9 @@
                     }
                 }
             }
+        },
+        "webCompat": {
+            "state": "disabled"
```

Comment:
This isn't really needed but also it's the only place we're not explicitly enabling webCompat.